autotest/autotest.py

```diff
@@ -65,6 +68,19 @@ def debug(self):
 def ansi(code):
     return lambda s: "\x1b[{}m{}\x1b[0m".format(code, s)
 
+def wait_for_success(wait_for=None):
+    secs, timeout = 0, 30
+    while secs <= timeout:
+        time.sleep(1)
+        output = [x for x in subprocess.check_output(["tmux", "capture-pane", "-p"]).split("\n") if x]
```

Comment:
As discussed — we can make that much simpler by just checking that the last two characters are `\n$`